equinix/fabric_connection_schema.go

```diff
@@ -0,0 +1,840 @@
+			Description: "Deleted by User Email Address",
+		},
+		"deleted_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by Date and Time",
+		},
+	}
+}
+
+var createOrderRes = &schema.Resource{
+	Schema: createOrderSch(),
+}
+
+func createOrderSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"purchase_order_number": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Purchase order number",
+		},
+		"billing_tier": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Billing tier for connection bandwidth",
+		},
+		"order_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Identification",
+		},
+		"order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Reference Number",
+		},
+	}
+}
+
+var createAccountRes = &schema.Resource{
+	Schema: createAccountSch(),
+}
+
+func createAccountSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"account_number": {
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Description: "Account Number",
+		},
+		"account_name": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Account Name",
+		},
+		"org_id": {
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Description: "Customer organization identifier",
+		},
+		"organization_name": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Customer organization name",
+		},
+		"global_org_id": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Global organization identifier",
+		},
+		"global_organization_name": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Global organization name",
+		},
+		"global_cust_id": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Global Customer organization identifier",
+		},
+	}
+}
+
+func createNotificationSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"type": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Notification Type",
+		},
+		"send_interval": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Send interval",
+		},
+		"emails": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Description: "Array of contact emails",
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+		},
+	}
+}
+
+func createErrorAdditionalInfoSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"property": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Property at which the error potentially occurred",
+		},
+		"reason": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Reason for the error",
+		},
+	}
+}
+
+func createOperationalErrorSch() map[string]*schema.Schema {
```

Comment:
duplicated from readOptionalErrorSch

Reply:
Yes we could reuse, I thought of separating these two resources to the entirety. I will catchup with you.